src/main/resources/org/jenkinsci/plugins/workflow/steps/ToolStep/help-name.html

```diff
@@ -0,0 +1,3 @@
+<div>
+ The name of the tool. The tool name must be pre-configured in Jenkins under Manage Jenkins → Global Tool Configuration.
```

Comment:
Add hyperlink to the management page? Or highlight the Web UI elements via italic or whatsover?

Reply:
Thanks @oleg-nenashev, I have made it italic